### PR TITLE
Update metrics

### PR DESCRIPTION
**Pull request by Ashparshp**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 65ba0c34386fd39be035cd166da7feb4765828c5 at: https://66aabf32521aed299f9ece00--layer5.netlify.app

File: src/collections/programs/lfx-2024/lfx-2024.mdx [hunk 1]
```diff
@@ -59,9 +59,9 @@ Part-time terms will start on the same schedule and last six versus three months
 
 <h3>About our Community</h3>
  
-Layer5 is all about <Link to="/community">its community</Link> of contributors. We have designed an onboarding program customized to <Link to="/community/newcomers">meet newcomers</Link> where they’re at and developed an onboarding buddy program, <Link to="/community/meshmates">MeshMates</Link> with individuals dedicated to assisting contributors. Layer5 and Meshery have been around for a year and half and have a healthy, growing community of 800+ contributors. The layer5.io website itself is open source and created by 400+ of our contributors.
+Layer5 is all about <Link to="/community">its community</Link> of contributors. We have designed an onboarding program customized to <Link to="/community/newcomers">meet newcomers</Link> where they’re at and developed an onboarding buddy program, <Link to="/community/meshmates">MeshMates</Link> with individuals dedicated to assisting contributors. Layer5 and Meshery have been around for a year and a half and have a thriving community of 1900+ contributors. The layer5.io website itself is open source and created by more than 1000 of our contributors.
```
Comment (sudhanshutech) [inline src/collections/programs/lfx-2024/lfx-2024.mdx:62]: The line `around for a year and half `needs to be updated with more specific. // @vishalvivekm can you?

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by sudhanshutech** [comment]
@Ashparshp is this done ? Have you confirmed the metric by asking someone and are feedbacks incorporated

**Comment by Ashparshp** [comment]
@sudhanshutech, still waiting for a number from you..

**Comment by sudhanshutech** [comment]
> @sudhanshutech, still waiting for a number from you..

// @vishalvivekm

File: src/collections/programs/lfx-2024/lfx-2024.mdx (hunk 1 above)
Comment (leecalcote) [inline src/collections/programs/lfx-2024/lfx-2024.mdx:62]:
```suggestion
Layer5 is all about <Link to="/community">its community</Link> of contributors. We have designed an onboarding program customized to <Link to="/community/newcomers">meet newcomers</Link> where they’re at and developed an onboarding buddy program, <Link to="/community/meshmates">MeshMates</Link> with individuals dedicated to assisting contributors. The Layer5 community and its open source projects are thriving.
```

**Pull request merged**